### DISCUSSION TITLE
adding control planes to existing cluster from different subnet

### DISCUSSION THREAD

**Discussion by matjazantloga** [siderolabs/talos#9192] [Q&A]
hi

i have running cluster in 10.99.100.x
i also have another location connected via ipsec , 10.99.202.x
when i try to add node from that subnet it doesn't work i'm not sure whether scenario like this is possible aka adding nodes via L3.

![image](https://github.com/user-attachments/assets/6f782049-8449-4b27-8402-9e807b63ad90)

on fw I added additional services 
![image](https://github.com/user-attachments/assets/3219d149-bdf0-4ecf-87c8-92ddb18fef68)

i also added kubespan
![image](https://github.com/user-attachments/assets/a5514246-468a-491c-9c1f-8024adb84b2c)

i assume main issue is that pod and service subnets are not routable via subnets and this is mission no go?

**Comment by smira** [comment]
Use https://www.talos.dev/v1.7/talos-guides/network/kubespan/#kubespanpeerstatuses to see KubeSpan status first to see that all machines are connected.
